API/API/DataAccessLayer/Repositories/UserRepository.cs

```diff
@@ -0,0 +1,65 @@
+﻿using API.DataAccessLayer.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace API.DataAccessLayer.Repositories
+{
+    public class UserRepository : IRepository<User>
```

Comment:
you should define abstract class BaseRepository and implement all CRUD operations with database there. inherit from BaseRepository class in UserRepository